src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -320,3 +339,90 @@ Here's an example of adding the `crossorigin="anonymous"` attribute:
 ```
 </Collapser>
 </CollapserGroup>
+
+## Use Nerdgraph to configure replay settings [#nerdgraph-mutation]
+
+You can configure session replay settings using our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) instead of using the UI. This method is helpful when...
```

Comment:
```suggestion
You can configure session replay settings using our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) instead of using the UI. This method is helpful if you need to make bulk changes, such as applying sampling rates to multiple browser entities.
```